n3fit/src/n3fit/backends/keras_backend/multi_dense.py

```diff
@@ -0,0 +1,163 @@
+from typing import List
+
+import tensorflow as tf
+from tensorflow.keras.initializers import Initializer
+from tensorflow.keras.layers import Dense
+
+
+class MultiDense(Dense):
+    """
+    Dense layer for multiple replicas at the same time.
+
+    For the first layer in the network, the input shape is (batch_size, gridsize, features),
+    still without a replica axis.
```

Comment:
```suggestion
    For architectures in which the input is shared across replicas, it is useful to have an input shape
    wihtout a replica axis, e.g., `(batch_size, gridsize, features)`. The `is_first_layer` argument
    can be used so that a replica axis is created upon output.
```

Reply:
What was your intention for this change? To me it makes it less clear, and downplays the impact of this argument, as it actually changes the operation the layer does (from applying all weights to the input, to applying slice i of the weights along the replica axis to slice i of the inputs along the replica axis), it's only this trivial in the single replica case. I updated the documentation to make this a bit more clear, let me know what you think.